minitwit_admin/src/components/TwitListComponent.vue

```diff
@@ -94,21 +100,81 @@ export default {
   components: {},
   emits: ["onClick"],
   setup(props, context) {
-    const handleItemClick = (item) => {
-      context.emit("onClick", item);
-    };
+    const store = inject("store");
+    const { getFollowers } = useFollowers();
+    const { getLoggedInUser } = useUsers();
+    const followers = getFollowers();
+    const loggedInUser = getLoggedInUser();
+
+    const isAlreadyFollowed = (authorId) => {
+      return followers.value.some(entry => entry.whomId === authorId)
+    }
+
+    const handleItemClick = (item) => context.emit("onClick", item);
+    const followUser = (userId) => store.followers.actions.followUser(userId);
```

Comment:
So you could emit something like "primaryActionButtonClick"
```suggestion
    const handlePrimaryActionButtonClicked = (item) => context.emit("onPrimaryActionButtonClick", item);
```